friends/src/App.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Header from "./components/Header/Header";
+import SearchFriend from "./components/SearchFriend/SearchFriend";
+import AddEditFriend from "./components/AddEditFriend/AddEditFriend";
+import FriendsList from "./components/FriendsList/FriendsList";
+import "./App.css";
+
+function App() {
+  const [friendList, setFriendList] = useState([]);
+  const [friendsToDisplay, setFriendsToDisplay] = useState([]);
+  const [editableFriend, setEditableFriend] = useState(undefined);
+  const URL = "http://localhost:5000/friends";
+
+  const getFriends = async () => {
+    try {
+      const friendsData = await axios.get(URL);
+      setFriendList(friendsData.data);
+      setFriendsToDisplay(friendsData.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const addNewFriend = async friend => {
+    try {
+      await axios.post(URL, friend);
+      await getFriends();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const updateFriend = async friend => {
+    try {
+      console.log(`${URL}/${friend.id}`);
```

Comment:
Remember to clean your code from pointless console logs.
Also good job with the dynamic path here.